.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           - os: ubuntu-latest
             python: 3.10
             docsTarget: true
+            protoCheckTarget: true
```

Comment:
I did this in TS and TBH it's probably better to just have a separate build for docs and proto check to avoid delaying the entire pipeline.

Reply:
I think it's fast enough, and it's on the fastest target so still gets done before mac/windows